tests/production/CloudStatusProductionTests.js

```diff
@@ -75,6 +78,8 @@ gpii.tests.productionConfigTesting.testDefs = fluid.transform(gpii.tests.develop
     return testDef;
 });
 
+fluid.log("gpii.tests.productionConfigTesting.testDefs[0].components = '" + JSON.stringify(gpii.tests.productionConfigTesting.testDefs[0].components, null, "\t") + "'");
```

Comment:
Please remove this line for debugging.

Reply:
Thanks for catching that.

Due to debugging these tests in GCP, I temporarily removing the login/logout tests.  It looks like I only put those tests back in but didn't completely revert the `expect` property.